src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java

```diff
@@ -161,20 +152,19 @@ public boolean updateUser() {
             int affectedRows = stmt.executeUpdate();
             return affectedRows > 0;
 
-            } catch (Exception ex) {
-           LOGGER.error("Error updating user with ID " + user.getId() + ": " + ex.getMessage());
+        } catch (Exception ex) {
+            LOGGER.error("Error updating user with ID " + user.getId() + ": " + ex.getMessage());
             ex.printStackTrace();
             return false;
         }
     }
 
     @Override
     public User getUsername(String username) {
-        String selectByIdQuery = "SELECT * FROM login WHERE username=?";
         User login = null;
 
         try (Connection connection = ConnectionHelper.getConnection();
-             PreparedStatement preparedStatement = connection.prepareStatement(selectByIdQuery)) {
+             PreparedStatement preparedStatement = connection.prepareStatement(GET_USERNAME_STATEMENT)) {
```

Comment:
fix the try-with-resource. remove preparedstatement from try()

Reply:
okay napo sir